### PR TITLE
#20300

### PR DESCRIPTION
**Pull request by Triplelexx**
changes regarding worklist #20300

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

File: interface/src/ui/ApplicationOverlay.cpp [hunk 1]
```diff
@@ -874,26 +873,29 @@ void ApplicationOverlay::renderAudioMeter() {
...
-    } else {
-        quadColor = glm::vec4(1, 1, 1, 1);
-    }
-    // Draw Blue (low level) quad
-    DependencyManager::get<GeometryCache>()->renderQuad(AUDIO_METER_X + AUDIO_METER_INSET, 
-                                                        audioMeterY + AUDIO_METER_INSET, 
-                                                        audioLevel, AUDIO_METER_HEIGHT - AUDIO_METER_INSET, quadColor,
-                                                        _audioBlueQuad);
+	if (audioLevel >= 0) {
+		glm::vec4 quadColor;
```
Comment (ZappoMan) [inline interface/src/ui/ApplicationOverlay.cpp]: coding standard: indents should be 4 spaces. Not TAB characters.

Comment (Triplelexx) [inline interface/src/ui/ApplicationOverlay.cpp]: Sorry I missed that, noted.

File: interface/src/ui/ApplicationOverlay.cpp [hunk 2]
```diff
@@ -874,26 +873,29 @@ void ApplicationOverlay::renderAudioMeter() {
...
-    glm::vec4 quadColor;
-    if (!isClipping) {
-        quadColor = AUDIO_METER_BLUE;
-    } else {
-        quadColor = glm::vec4(1, 1, 1, 1);
+    if (audioLevel >= 0) {
+        glm::vec4 quadColor;
+        if (!isClipping) {
+            quadColor = AUDIO_METER_BLUE;
+        }
```
Comment (ZappoMan) [inline interface/src/ui/ApplicationOverlay.cpp]: sorry, I missed this. This is also out of sync with the coding standard... should be...

```
} else {
```

**Comment by birarda** [comment]
ok to test

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2f942c02637ecf9cab12b1514e94d7c8717c72f available at: https://gist.github.com/01f51c3403e525a5a954<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4265.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4265.dmg

**Comment by birarda** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2f942c02637ecf9cab12b1514e94d7c8717c72f available at: https://gist.github.com/adb05553dc52cefd3a7e<br/><br/>Installers available for download at:<br>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4265.exe<br>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4265.dmg

**Pull request merged**